src/metabase/query_processor/middleware/parameters/mbql.clj

```diff
@@ -117,5 +117,6 @@
 
       :else
       (let [filter-clause (build-filter-clause query (assoc param :value param-value))
-            query         (mbql.u/add-filter-clause query filter-clause)]
+            [_ _ opts]    target
```

Comment:
In fact, this seems to be taking the opts from the dimension clause, not from the field clause, which is good.